README.md

```diff
@@ -38,6 +38,27 @@ To run the tests, you also must have `nose` installed.
 
 ### User Installation
 
+#### Fedora Release 25 onwards
+
+Install Pomegranate and its dependencies ( for Python3 )
+
+```
+$ sudo dnf install -y python3-virtualenv python3-scipy python3-numpy python3-networkx python3-Cython
+$ virtualenv env3 --python=python3.5 --system-site-packages
+$ source env3/bin/activate
+(env3) $ pip3 install wheel nose cython numpy scipy networkx
+(env3) $ pip3 install --no-cache-dir pomegranate
+```
+
+Our virtualenv is ready, so now we can check that the installation is ready
+
+```
+(env3) $ python -c 'from pomegranate import *; print("It works")'
+It works
+```
+
+#### Other OSes
```

Comment:
If the `Fedora Release 25 Onward` is moved after this, I don't think this line is needed.